test/integration/tarantool_role_test.lua

```diff
@@ -0,0 +1,165 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local helpers = require('test.helper')
+local Server = require('test.helper_server')
+
+local g = t.group('tarantool_role_integration_test')
+
+g.before_all(function (cg)
+    t.skip_if(not helpers.tarantool_role_is_supported(),
+            'Tarantool role is supported only for Tarantool starting from v3.0.0')
+
+    local test_dir = fio.tempdir()
+    cg.router = Server:new({
+        config_file = fio.abspath(fio.pathjoin('test', 'integration', 'simple_app', 'config.yaml')),
+        env = {LUA_PATH = helpers.lua_path},
+        chdir = fio.pathjoin(test_dir, 'simple_app'),
+        alias = 'master',
+        workdir = fio.pathjoin(test_dir, 'simple_app'),
+    })
+end)
+
+g.before_each(function(cg)
+    fio.mktree(cg.router.workdir)
+    --fio.copytree(fio.abspath('./.rocks'), fio.pathjoin(cg.router.workdir, '.rocks'))
+    cg.router:start{wait_until_ready = true}
```

Comment:
We start instance before each test because
we need to force reload of expirationd role and also instance environment
from previous tests can influence test result.
(e.g function creation, when testing that role doesn't start w/o it)
Restarting instance is the easiest way to achieve it.
It takes around 1s to start an instance, which considering small amount of integration tests is not a problem.

Reply:
Ok, it's fine then.